src/lxterminal.c

```diff
@@ -451,24 +451,23 @@ static void terminal_close_tab_activate_event(GtkAction * action, LXTerminal * t
  * Close the current window. */
 static void terminal_close_window_activate_event(GtkAction * action, LXTerminal * terminal)
 {
-    int close_flag = 0;
-    if (terminal->terms->len > 1)
+    int close_flag = 1;
+    if (!get_setting()->disable_confirm)
     {
-        GtkWidget * dialog = gtk_message_dialog_new(
-                GTK_WIDGET(terminal->window), GTK_DIALOG_MODAL, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO,
-                "You are about to close %d tabs. Are you sure you want to continue?", terminal->terms->len);
-        gtk_window_set_title(GTK_WINDOW(dialog), "Confirm close");
-        gint result = gtk_dialog_run(GTK_DIALOG(dialog));
-        gtk_widget_destroy(dialog);
-        if (result == GTK_RESPONSE_YES)
+        if (terminal->terms->len > 1)
         {
-            close_flag = 1;
+            GtkWidget * dialog = gtk_message_dialog_new(
+                    GTK_WIDGET(terminal->window), GTK_DIALOG_MODAL, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO,
+                    "You are about to close %d tabs. Are you sure you want to continue?", terminal->terms->len);
```

Comment:
Please make this string translatable by wrapping `_()` around the string. Like _("Hello World")